Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterWindow.cpp

```diff
@@ -842,23 +841,28 @@ namespace EMStudio
             return;
         }
 
-        ParameterCreateEditDialog* createEditParameterDialog = new ParameterCreateEditDialog(m_plugin, this);
-        createEditParameterDialog->Init();
+        if (!m_parameterCreateEditWidget)
+        {
+            m_parameterCreateEditWidget = new ParameterCreateEditWidget(m_plugin, this);
+            m_parameterCreateEditWidget->Init();
+        }
+        else
+        {
+            m_parameterCreateEditWidget->Reinit(nullptr);
+            QObject::disconnect(m_parameterCreateEditWidget, &EMStudio::ParameterCreateEditWidget::accept, nullptr, nullptr);
+        }
 
-        EMStudio::ParameterCreateEditDialog::connect(createEditParameterDialog, &QDialog::finished, [=](int resultCode)
+        EMStudio::InspectorRequestBus::Broadcast(&EMStudio::InspectorRequestBus::Events::Update, m_parameterCreateEditWidget);
+        EMStudio::ParameterCreateEditWidget::connect(m_parameterCreateEditWidget, &EMStudio::ParameterCreateEditWidget::accept, [=]()
```

Comment:
suggestion: Capturing explicit values as opposed to using the blanket `=` is generally preferred to help avoid any costly copies and to make it clear what variables from the outside scope are being used